examples/tv-casting-app/tv-casting-common/core/CastingPlayer.h

```diff
@@ -36,6 +38,16 @@ const int kPortMaxLength = 5; // port is uint16_t
 const int kIdMaxLength                                      = chip::Dnssd::kHostNameMaxLength + kPortMaxLength + 1;
 const unsigned long long int kCommissioningWindowTimeoutSec = 3 * 60; // 3 minutes
 
+/**
+ * @brief Describes an Endpoint that the client wants to connect to
+ */
+struct EndpointFilter
+{
+    uint16_t vendorId  = 0;
```

Comment:
does 0 mean not-specified?

Reply:
Yes. I've added documentation to clarify that.